go/common/rpc/generated/enclave_grpc.pb.go

```diff
@@ -1,7 +1,7 @@
 // Code generated by protoc-gen-go-grpc. DO NOT EDIT.
```

Comment:
are these changed because they were generated with a newwer version?
Will that have any impact?

Reply:
Yeah good spot.
I regenerated the proto because I dropped the enclave method that I had previously added.
But I can just take the existing protoc, should be ok.

You do raise a very valid point though, seems like I'm running a different version, we should do a version lock on this to avoid possible future issues.